app/controllers/api/projects.php

```diff
@@ -395,6 +398,7 @@
                 ->setAttribute('legalCity', $legalCity)
                 ->setAttribute('legalAddress', $legalAddress)
                 ->setAttribute('legalTaxId', $legalTaxId)
+                ->setAttribute('authDuration', $authDuration * 60)
```

Comment:
Same as the comment above. Replicate the behavior of previous auth related routes in the projects service.